contracts/testing/MetaPoolMock.vy

```diff
@@ -102,6 +102,12 @@ def get_dy(i: int128, j: int128, dx: uint256) -> uint256:
     return self._get_dy(self.coin_list[i], self.coin_list[j], dx)
 
 
+@external
+@view
+def A_precise() -> uint256:
+    return self.A
```

Comment:
Actually, in real pools `A_precise` is larger than `A` by a multiplier `A_PRECISION = 100`